src/MOI_wrapper.jl

```diff
@@ -877,14 +883,70 @@ function MOI.delete(
     return
 end
 
+"""
+    _set_variable_lower_bound(model, info, value)
+
+This function is used to indirectly set the lower bound of a variable.
+
+We need to do it this way to account for potential lower bounds of 0.0 added by
+VectorOfVariables-in-SecondOrderCone constraints.
+
+See also `_get_variable_lower_bound`.
+"""
+function _set_variable_lower_bound(model, info, value)
+    if info.num_soc_constraints == 0
+        # No SOC constraints, set directly.
+        @assert isnan(info.lower_bound_if_soc)
+        set_dblattrelement!(model.inner, "LB", info.column, value)
+        _require_update(model)
+    elseif value >= 0.0
+        # Regardless of whether there are SOC constraints, this is a valid bound
+        # for the SOC constraint and should over-ride any previous bounds.
+        info.lower_bound_if_soc = NaN
+        set_dblattrelement!(model.inner, "LB", info.column, value)
+        _require_update(model)
+    elseif isnan(info.lower_bound_if_soc)
+        # Previously, we had a +ve lower bound (i.e., it was set in the case
```

Comment:
It's better to write out positive and negative. It took a few seconds for me to figure out what +ve and -ve meant.